Neos.Neos/Classes/FrontendRouting/Projection/DocumentUriPathFinder.php

```diff
@@ -28,7 +28,7 @@ final class DocumentUriPathFinder implements ProjectionStateInterface
 
     public function __construct(
         private readonly Connection $dbal,
-        private readonly string $tableNamePrefix
+        private readonly string $tableNamePrefix,
```

Comment:
youre ide seems pretty agressive. its hard to see in this diff the actual changes  😅